common/common.mk

```diff
@@ -17,7 +17,8 @@
 CONTAINERD_REMOTE ?=https://github.com/containerd/containerd.git
 RUNC_REMOTE       ?=https://github.com/opencontainers/runc.git
 REF?=HEAD
-RUNC_REF?=dc9208a3303feef5b3839f4323d9beb36df0a9dd
+# v1.0.0-rc91
+RUNC_REF?=24a3cf88a7ae5f4995f6750654c0e2ca61ef4bb2
```

Comment:
Do you know if this bump will be backported to the v1.3 branch?

Reply:
Probably yes